Source/DafnyLanguageServer/Language/Symbols/SymbolTable.cs

```diff
@@ -41,14 +44,16 @@ public class SymbolTable {
         IDictionary<AstElement, ILocalizableSymbol> declarations,
         IDictionary<ISymbol, SymbolLocation> locations,
         IIntervalTree<Position, ILocalizableSymbol> lookupTree,
-        bool symbolsResolved
+        bool symbolsResolved,
+        ILogger iLogger
```

Comment:
`ILogger<SymbolTable> logger`
See note in `TextDocumentLoader`. For consistency, the `logger` parameter should be the first of the constructor.